### PR TITLE
15: add action to run tests on FireBase

### PR DESCRIPTION
**Pull request by greenEkatherine**
Addressed #15

### CODE REVIEW

File: .github/workflows/firebase.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+        uses: google-github-actions/setup-gcloud@master
+        with:
+          version: '290.0.1'
+          service_account_key: ${{ secrets.FIREBASE_KEY }}
+          project_id: ${{ secrets.PROJECT_ID }}
+
+      - name: Run Instrumentation Tests in Firebase Test Lab
+        run: |
+          gcloud firebase test android models list
+          gcloud firebase test android run --type instrumentation --num-flaky-test-attempts 3 --use-orchestrator --app app-debug/app-debug.apk --test app-debug-androidTest/app-debug-androidTest.apk --device model=Pixel2,version=28,locale=en,orientation=portrait
```
Comment (omen273) [inline .github/workflows/firebase.yml:64]: Is it possible to use strings less then 100 symbols? Because it was a problem for me to find android version:)

**Review by omen273 — CHANGES_REQUESTED** [review]

File: .github/workflows/firebase.yml [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+      - uses: actions/checkout@v2
+      - name: set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+
+      - name: Build APK for UI test after Unit tests
+        run: |
+          ./gradlew test
+          ./gradlew assembleDebug
```
[the remark this answers is not in the document]
Comment (greenEkatherine) [reply .github/workflows/firebase.yml:24]: We've already discussed this offline that I was not able quickly switch to release mode. That's why it was moved into the separate issue #52 and I will solve it there

File: .github/workflows/firebase.yml [hunk 3]
```diff
@@ -0,0 +1,64 @@
+name: FireBase
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    # run every day at 01:00
+    - cron:  '0 1 * * *'
```
[the remark this answers is not in the document]
Comment (greenEkatherine) [reply .github/workflows/firebase.yml:7]: It should be UTC

File: .github/workflows/firebase.yml (hunk 1 above)
Comment (greenEkatherine) [reply .github/workflows/firebase.yml]: Done

File: .github/workflows/firebase.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
+name: FireBase
+
+on:
+  schedule:
+    # * is a special character in YAML so you have to quote this string
```
[the remark this answers is not in the document]
Comment (greenEkatherine) [reply .github/workflows/firebase.yml]: Done

File: .github/workflows/firebase.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (greenEkatherine) [reply .github/workflows/firebase.yml]: Removed

File: .github/workflows/firebase.yml (hunk 3 above)
[the remark this answers is not in the document]
Comment (omen273) [reply .github/workflows/firebase.yml:6]: What do you mean? https://en.wikipedia.org/wiki/UTC%C2%B100:00?

File: .github/workflows/firebase.yml [hunk 5]
```diff
@@ -0,0 +1,65 @@
+name: FireBase
+
+on:
+  schedule:
+    # run every day at 01:00
```
Comment (omen273) [inline .github/workflows/firebase.yml:5]: Please add timezone in the comment

**Review by omen273 — APPROVED** [review]

File: .github/workflows/firebase.yml (hunk 5 above)
Comment (greenEkatherine) [reply .github/workflows/firebase.yml]: Done

**Pull request merged**